### ISSUE TITLE
MBP: My Backpack Updates SF Contact create date

### ISSUE THREAD

**Issue by WaBirder57** [washingtontrails/vms#471]
@mscholtz Would you expect that the MyBackpack integration with Salesforce would update the SF Contact Create Date field?  For example, I logged in this morning to MyBackpack (now live!) on the WTA website and I see that the Create Date field on my contact record now shows today date:  https://wta.my.salesforce.com/003A000001hYd1t  Seems like Create Date should only show to initial date that a contact record was created.

**Comment by davisagli** [comment]
No, if the creation date is today, then that contact was created today. Is there another contact it was supposed to match?

**Comment by davisagli** [comment]
I don't see any other contacts with Website_Member_ID__c = 'CharlieKahle' or any with FirstName = 'Charlie', LastName = 'Kahle' and Email='charles@wta.org' so I think it's correct per our matching rules that it created a new contact instead of matching an existing one.

**Comment by WaBirder57** [comment]
OK I guess that makes sense.  Birthdate is being set to a default value of 1/1/1800, is that set via MBP or in Salesforce?

**Comment by mscholtz** [comment]
Huh, that last part is odd.  Don't think that's an SF thing.  David, are you making sure not to pass the birthdate if one isn't entered?

**Comment by davisagli** [comment]
Hmm, looks like we have that default in Plone. @alecpm do you remember why? If it needs to stay, we can adjust the sync code so that it passes an empty value if the date is 1/1/1800

**Comment by davisagli** [comment]
A blank birthdate should now get sent to Salesforce correctly.
